packages/components/psammead-section-label/src/index.test.jsx

```diff
@@ -15,7 +15,7 @@ describe('SectionLabel', () => {
 
       shouldMatchSnapshot(
         'should render correctly with explicitly showing the bar',
-        <SectionLabel script={latin} labelId="test-section-label" bar>
+        <SectionLabel script={latin} labelId="test-section-label">
```

Comment:
Howcome the `bar` was removed? The title of `should render correctly with explicitly showing the bar` is no longer true as its not `explicit`

Reply:
Sorry about that. Returning it